reana_cluster/cli/cluster.py

```diff
@@ -68,6 +69,34 @@ def get(ctx, component, namespace):
         logging.debug(str(e))
 
 
+@click.command(help='Configure your shell environment with REANA components'
```

Comment:
components'

Reply:
The `env` command only produces the commands how people can customise their environments, so it would be better to say:

     Display commands to set up the environment for the REANA client 

Emulating `docker-machine` quite closely.  I think that would be more accurate...